src/03-modality-agnostic-files.md

```diff
@@ -452,6 +452,8 @@ For example, the [EDF](https://www.edfplus.info/)
 data format can only contain recording dates after 1985.
 Shifting dates is RECOMMENDED, but not required.
 
+The source file path of each recording is also OPTIONAL and can be stored under the ``sources`` column. Similar to [derivatives](./05-derivatives/02-common-data-types.md), it is OPTIONAL to include ``sources``, specifying the filename(s) of the source file used to generate this dataset. If the filename(s) contains patient identifiable information, then it should not be stored.
```

Comment:
double backticks render code in RST, but for MD, it's single backticks :wink: 

Also: the files under the `filename` column are relative to the `scans.tsv` file (this is implicit unfortunately, but relatively clear from the context that this is meant by the text on line 435). --> What should the files under the `sources` column be relative to? ... should we wait for #918 and make this BIDS-URIs? ... or should it be relative to the dataset root - as you seem to do in your current example?